community/security/src/main/java/org/neo4j/commandline/admin/security/SetPasswordCommand.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.commandline.admin.security;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.neo4j.commandline.admin.AdminCommand;
+import org.neo4j.commandline.admin.CommandFailed;
+import org.neo4j.commandline.admin.IncorrectUsage;
+import org.neo4j.graphdb.factory.GraphDatabaseSettings;
+import org.neo4j.kernel.configuration.Config;
+import org.neo4j.logging.NullLog;
+import org.neo4j.logging.NullLogProvider;
+import org.neo4j.server.security.auth.BasicAuthManager;
+import org.neo4j.server.security.auth.BasicPasswordPolicy;
+import org.neo4j.server.security.auth.FileUserRepository;
+import org.neo4j.server.security.auth.PasswordPolicy;
+
+import static java.time.Clock.systemUTC;
+import static org.neo4j.commandline.admin.security.DatabaseManagementSystemSettings.auth_store_directory;
+
+public class SetPasswordCommand implements AdminCommand
+{
+    public class Provider extends AdminCommand.Provider
+    {
+
+        protected Provider()
+        {
+            super( "set-password" );
+        }
+
+        @Override
+        public Optional<String> arguments()
+        {
+            return Optional.of( "<username> <password>" );
+        }
+
+        @Override
+        public String description()
+        {
+            return "Sets the password for the specified user and activates that user in case it is suspended";
```

Comment:
`activates that user` what does this mean? Are you referring to the fact that `password_change_required` is cleared?